src/components/Popover/__tests__/Popover.test.tsx

```diff
@@ -0,0 +1,291 @@
+import React from 'react';
+import {fireEvent, render, screen} from '@testing-library/react';
+import user from '@testing-library/user-event';
+
+import '../../../tests/utils/fakeTimers';
+import {Popover} from '../Popover';
+import type {PopoverProps} from '../';
+
+const defaultTooltipContent = 'Tooltip';
+const defaultTriggerText = 'Trigger';
+
+const renderPopover = (props?: Partial<PopoverProps>) => (
+    <Popover content={defaultTooltipContent} qa="popover" openOnHover={false} {...props}>
+        {defaultTriggerText}
+    </Popover>
+);
+
+const waitForTooltipOpenedStateChange = () => jest.advanceTimersByTime(300);
+
+const checkIfPopoverOpened = () => {
+    const popover = screen.queryByTestId('popover-tooltip');
+
+    expect(popover).toHaveClass('yc-popup_open');
```

Comment:
may be we can replace `toHaveClass` with `toBeVisible`?

Reply:
No, it's  always visible for jest.

Reply:
Even when there is no `open` modifier?

Reply:
Yes. It's in the document and visible for jest. Maybe it's something with styles.

Reply:
Do you actually call this function when `<Popover>` is not shown?

Reply:
Yes